src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -27,6 +27,8 @@
  * Represents the in-memory model of the address book data.
  */
 public class ModelManager implements Model {
+    private int currentContactId = 0;
```

Comment:
You could consider renaming to latestContactId? I think the name can be clearer about the intention of this variable.

Reply:
Renamed as suggested!